modules/meter/main/index.coffee

```diff
@@ -33,13 +33,15 @@ class Meter extends hx.EventEmitter
       trackerBackgroundCol: hx.color(hx.theme.palette.contrastCol).alpha(0.05).toString()
       markerCol: hx.theme.palette.contrastCol
       valueFormatter: (value, isTotal) -> if isTotal then "#{hx.userFacingText('meter', 'of')} #{value}" else value
+      redrawOnResize: true
```

Comment:
I think this probably warrants making it a `Minor` release but I also can't decide if it could go in with a `Patch`?